components/work_templates/index.tsx

```diff
@@ -134,13 +145,35 @@ const WorkTemplateModal = () => {
         setSelectedVisibility(visibility);
     };
 
-    const create = (template: WorkTemplate, name = '') => {
-        const str = `TODO: creating elements from template ${template!.id} with name ${name || '<empty>'}`;
+    const create = async (template: WorkTemplate, name = '', visibility = Visibility.Public) => {
+        const pbTemplates = template.content.filter((item) => item.playbook).map((item) => {
+            return playbookTemplates.find((pb) => pb.title === item.playbook!.template)!;
+        });
+
+        const req: ExecuteWorkTemplateRequest = {
+            team_id: teamId,
+            name,
+            visibility,
+            work_template: template,
+            playbook_templates: pbTemplates,
+        };
 
-        // @TODO REMOVE THIS FOR PROD
-        // eslint-disable-next-line no-alert
-        alert(str);
+        setIsCreating(true);
+        const {data, error} = await dispatch(executeWorkTemplate(req)) as ActionResult<ExecuteWorkTemplateResponse>;
 
+        if (error) {
+            setIsCreating(false);
```

Comment:
apart from enabling the button, probably we could also inform the user about the problem while creating the work templates

Reply:
Good point, the discussions are still "pending" on how to manage errors right now but I should add a @TODO here to make this clear.